services/QuillLMS/app/services/classlink_integration/instructor_linker.rb

```diff
@@ -11,17 +11,25 @@ def initialize(persisted_user:, classlink_instance_id:, school_sourced_id:)
     end
 
     def run
-      return unless persisted_user.role == ::User::ADMIN
+      return unless [::User::ADMIN, ::User::TEACHER].include?(persisted_user.role)
```

Comment:
I think this is a law of demeter violation.  Instead of reaching into the class for constants, use an existing method:

`return unless persisted_user.teacher?`  (This includes `admin?`)